examples/v2/chat2.nim

```diff
@@ -303,6 +304,11 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
 
     await node.query(HistoryQuery(contentFilters: @[HistoryContentFilter(contentTopic: DefaultContentTopic)]), storeHandler)
 
+  if conf.swap:
+    for peerId, bal in node.wakuSwap.accounting:
+      debug "Accounting: ", peer=peerId, balance=bal
```

Comment:
This logs the balances on startup of the `chat2` client (only). `chat2` is merely a CLI application used to demonstrate/play around with `wakunode2` and is not, for example, what is running on the fleet nodes.  If you want this log during startup for all `wakunode2`s, you may want to move it somewhere [here](https://github.com/status-im/nim-waku/blob/master/waku/v2/node/wakunode2.nim#L667-L668)

Reply:
Ohh... I get it now. So chat2 was solely created as a CLI client for the purpose of testing out wakunode2?.
Thanks for the clarification. I always thought chat2 was part of the core implementation. I'll make the changes now.

Reply:
What @jm-clius is saying is definitely correct, I might've pointed you in this direction @EbubeUd in terms of a user story to know when acceptance criteria is met. "When running chat2, I should be able to see accounting state". Do also note that I believe some accounting state is printed in credit/debit function of swap...